### DISCUSSION TITLE
How to system beep?

### DISCUSSION THREAD

**Discussion by ruifortes** [denoland/deno#11189] [Q&A]
It would be nice if it was a real system beep and not a sound file

already tried:  `console.log('\007')` and `Deno.stdout.write('\x07')` and... not even a pip :-(

**Comment by justinmchase** [comment]
This worked for me:

```ts
$ deno
Deno 1.11.2
exit using ctrl+d or close()
> console.log('\x07')

undefined
```

**Reply by ruifortes** [reply]
That worked here also but not if I put it in file.
Maybe something to do with security? Although I run it with -A

**Reply by justinmchase** [reply]
Did you try `\b` I thought that was the classic bell sound escape character but both node and deno seemed to expect `\x07` from the repl.
